pkg/cloudprovider/providers/azure/azure_loadbalancer.go

```diff
@@ -273,6 +284,11 @@ func (az *Cloud) selectLoadBalancer(clusterName string, service *v1.Service, exi
 				Location:                     &az.Location,
 				LoadBalancerPropertiesFormat: &network.LoadBalancerPropertiesFormat{},
 			}
+			if az.useStandardLoadBalancer() {
```

Comment:
This will not be true now due to ' wantLb && !az.useStandardLoadBalancer()'

Reply:
ack